packages/graphql/tests/tck/aggregations/where/edge/duration.test.ts

```diff
@@ -69,7 +69,7 @@ describe("Cypher Aggregations where edge with Duration", () => {
             CALL {
                 WITH this
                 MATCH (this1:\`User\`)-[this0:LIKES]->(this)
-                RETURN any(var2 IN collect(this0.someDuration) WHERE var2 = $param0) AS var3
+                RETURN any(var2 IN collect(this0.someDuration) WHERE datetime() + var2 = datetime() + $param0) AS var3
```

Comment:
Is there a risk that the two separate calls of `datetime()` yield slightly different results, which could compromise the comparison? Is it worth having something like the below instead:

```suggestion
                WITH *, datetime() AS base
                RETURN any(var2 IN collect(this0.someDuration) WHERE base + var2 = base + $param0) AS var3
```

Reply:
There shouldn't be a risk here, the whole operation runs in a transaction and datetime uses the same value for the whole transaction.

As a test of that, the following cypher:

```cypher
WITH datetime() as d1
CALL apoc.util.sleep(1000)
RETURN d1, datetime()
```

RETURN the exact same time for d1 and the last datetime